src/pages/questionPage.js

```diff
@@ -45,6 +48,10 @@ export const initQuestionPage = () => {
             }
           }
         }
+        const answerLinks = currentQuestion.links.forEach((link) => {
```

Comment:
As I see `answerLinks` variable is not used anywhere. I think you don't need to assign this loop to a variable.

Reply:
done